date-input.html

```diff
@@ -9,8 +9,12 @@
 -->
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-input/iron-input.html">
-<link rel="import" href="../iron-flex-layout/classes/iron-flex-layout.html">
+<link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../iron-validatable-behavior/iron-validatable-behavior.html">
+<link rel="import" href="../paper-input/paper-input-container.html">
```

Comment:
There's mixins from this file that are being applied (like `@apply(--paper-input-container-font);`). It's not a big deal, since the parent including this element is already importing the container.